CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.23
+
+###### _Jan 14, 2021_
+
+Big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Support public paths in module augmentation (#24267) @eps1lon
+- 👩‍🎤 Migrate the Paper to emotion (#24397) @povilass
```

Comment:
```suggestion
- 👩‍🎤 Migrate the Paper and CssBaseline to emotion (#24397, #24176) @povilass @mnajdova 
```

Reply:
Also likely a good opportunity to mention the migration issue and that we seek for assistance

Reply:
I've extended this point with seeking assistance, please take a look